public/app/features/datasources/partials/http_settings.html

```diff
@@ -7,7 +7,7 @@ <h3 class="page-heading">HTTP</h3>
         <input class="gf-form-input" type="text"
               ng-model='current.url' placeholder="{{suggestUrl}}"
               bs-typeahead="getSuggestUrls"  min-length="0"
-              ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/" required></input>
+              ng-pattern="/^(ftp|http|https|unix):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/" required></input>
```

Comment:
Why removing the `&` character in regex?

Reply:
`&` not removed
Not sure why it gets highlighted